android/app/src/main/res/layout/activity_manual_control.xml

```diff
@@ -42,26 +42,38 @@
         app:layout_constraintTop_toTopOf="parent"></ImageView>
 
     <TextView
-        android:id="@+id/textView2"
+        android:id="@+id/speedHeader"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="24dp"
-        android:layout_marginLeft="24dp"
-        android:layout_marginTop="4dp"
-        android:text="Speed"
-        app:layout_constraintStart_toStartOf="@+id/joystickView2"
-        app:layout_constraintTop_toTopOf="@+id/joystickView2" />
+        android:layout_marginStart="4dp"
+        android:layout_marginLeft="4dp"
+        android:layout_marginBottom="8dp"
+        android:fontFamily="serif"
+        android:text="Speed:"
+        android:textColor="@color/teal_700"
+        android:textSize="20sp"
+        android:textStyle="bold"
+        app:layout_constraintBottom_toTopOf="@+id/joystickView2"
+        app:layout_constraintStart_toStartOf="@+id/imageView"
+        app:layout_constraintTop_toTopOf="@+id/imageView"
+        app:layout_constraintVertical_bias="1.0" />
```

Comment:
When testing, the speed and angle seem to still overlap the camera view screen. I think that these changes to lines 56-57 may fix this.
```suggestion
        app:layout_constraintBottom_toTopOf="@+id/joystickView2"
        app:layout_constraintStart_toStartOf="@+id/imageView"
        app:layout_constraintTop_toBottomOf="@+id/imageView" />
```

Reply:
I notice the issue now. I had added a constraint from the top to top of the camera view as instead it should have been to the bottom. You are correct nice catch!